docs/core/additional-tools/dotnet-coverage.md

```diff
@@ -594,8 +594,7 @@ If you don't want to use the `instrument` command, then the files to be instrume
 
 ```console
 D:\examples\ConsoleApp> dotnet-coverage collect --include-files .\bin\Debug\net7.0\*.dll dotnet run
```

Comment:
Since we're updating the article to .NET 9 SDK let's check all the paths to match.  I only commented on this one instance, but you should sweep the article for other occurrences outside the current diff:

```suggestion
D:\examples\ConsoleApp> dotnet-coverage collect --include-files .\bin\Debug\net9.0\*.dll dotnet run
```